### PR TITLE
Calculate positive (making CDC criteria) vs. negative (missing CDC Criteria) streaks for Criteria 1.

### LINKED ISSUE #6: Calculate streak of how long we've been meeting each CDC criteria
E.g., 1A has been TRUE for 14 days, or FALSE for 9 days, etc.

### PR DESCRIPTION
**Pull request by smacpher**

### CODE REVIEW

**Comment by smacpher** [comment]
So far, this I've only implemented this for Criteria 1 (A, B, C, D, Combined).

File: covid/transform.py [hunk 1]
```diff
@@ -127,6 +125,32 @@
 LAST_UPDATED_FIELD = "last_updated"
 STATE_FIELD = "State"
 
+# Define a pre-formatted string to be used for CDC criteria positive and negative streak fields.
+CDC_CRITERIA_POSITIVE_STREAK_FIELD_PRE_FORMAT = "{criteria_field} Positive Streak"
```
Comment (lucasmbrown) [inline covid/transform.py:146]: Nice!

File: covid/transform.py [hunk 2]
```diff
@@ -242,6 +268,7 @@
 
 
 def transform_covidtracking_data(covidtracking_df):
+    """Transforms data from https://covidtracking.com/ and calculates CDC Criteria 1 (A, B, C, D) and 2 (A, B, C, D)."""
```
Comment (lucasmbrown) [inline covid/transform.py:403]: 👍

File: covid/transform.py [hunk 3]
```diff
@@ -858,3 +920,17 @@ def indication_of_rebound(series_):
             indicator = "Rebound"
 
     return indicator
+
+
+def calculate_cdc_criteria_positive_and_negative_streak_series(criteria_series):
```
Comment (lucasmbrown) [inline covid/transform.py]: nit: should we move this to `transform_utils` and add a quick unit test? both are unnecessary, so don't worry about it if that seems over the top!

**Review by lucasmbrown — APPROVED** [review]
Left one comment about moving the util to the `transform_utils` file but otherwise looks great! 

Would love to implement this for the other criteria in a separate PR, too.

File: covid/transform.py (hunk 3 above)
Comment (ghost) [reply covid/transform.py]: Good idea! On it!

File: covid/transform_utils.py [hunk 4]
```diff
@@ -165,3 +165,18 @@ def calculate_state_summary(transformed_df, columns):
     ]
 
     return state_summary_df
+
+
+def calculate_consecutive_boolean_series(boolean_series):
```
Comment (lucasmbrown) [inline covid/transform_utils.py:170]: very nice! Appreciate the naming convention cleanup too.

File: README.md [hunk 5]
```diff
@@ -23,6 +23,11 @@ that Google service account and save it as `service-account-key.json` in the roo
 Next, add the "email address" of your service account to have permissions to the [Data Sources](https://drive.google.com/drive/u/1/folders/15j1iyyJtJ8BmK3y-HO6cLp-7R7nAoSml)
 folder of the Google Drive through the user interface which looks like this:
 
+Finally, make sure that the Google Sheets and Drive APIs are enabled it your service account's project. Go to these links:
```
Comment (lucasmbrown) [inline README.md:26]: Very nice!

**Review by lucasmbrown — APPROVED** [review]
Looks great!

**Pull request merged**